### ISSUE TITLE
[a11y] Firefox release notes page color contrast issue

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#15342]
# Axe-core® Accessibility Results for /en-US/firefox/releasenotes/

Page URL: https://www-dev.allizom.org/en-US/firefox/releasenotes/

## axe-core found 1 violation

| #   | Description | Axe rule ID | WCAG | Impact | Count |
| --- | --- | --- | --- | --- | --- |
| [1](#1) | Elements must meet minimum color contrast ratio thresholds | color-contrast | WCAG 2 Level AA, WCAG 1.4.3 | serious | 1   |

### Failed

1. Elements must meet minimum color contrast ratio thresholds

[Learn more](https://dequeuniversity.com/rules/axe/4.10/color-contrast?application=playwright)

color-contrast

WCAG 2 Level AA, WCAG 1.4.3

Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds

serious

Issue Tags: cat.color wcag2aa wcag143 TTv5 TT13.c EN-301-549 EN-9.1.4.3 ACT

| #   | Issue Description | To solve this violation, you need to... |
| --- | --- | --- |
| 1   | **Element location**<br><br>```<br>.all-download > a<br>```<br><br>**Element source**<br><br>```<br><a href="/en-US/firefox/all/desktop-release/">All Firefox downloads</a><br>``` | Fix any of the following:<br><br>*   Element has insufficient color contrast of 3.55 (foreground color: #0060df, background color: #cdcdd4, font size: 12.0pt (16px), font weight: bold). Expected contrast ratio of 4.5:1<br><br>**Related node:**<br><br>```<br>.all-download<br>``` |

**Comment by clarapenn** [comment]
Hi @alexgibson, I'm wondering whether a colour change here to address the color contrast ratio will interfere with set branding colours? Could you recommend what colour to swap it to instead? Or is there a style reference I should consult? Thanks so much!

**Comment by alexgibson** [comment]
@clarapenn I think I'd suggest changing the background color for this link from `$color-marketing-gray-30` to `$color-marketing-gray-20`.
